lib/oli_web/views/layout_view.ex

```diff
@@ -131,4 +132,55 @@ defmodule OliWeb.LayoutView do
   defp show_collab_space?(nil), do: false
   defp show_collab_space?(%CollabSpaceConfig{status: :disabled}), do: false
   defp show_collab_space?(_), do: true
+
+  def render_license(%{license_type: :custom} = assigns) do
+    ~H"""
+    <div
+      id="license"
+      class="container mx-auto flex flex-row px-10 items-center overflow-y-auto h-[40px] max-h-[40px] relative top-0"
+    >
+      <p class="h-full">License: <%= @custom_license_details %></p>
```

Comment:
Hardcoding "License: " here probably isn't a good idea.  The user can always include that string in the custom portion of the license.